source/clear-linux/virtual-environments/kvm.rst

```diff
@@ -3,60 +3,80 @@
 Running Clear Linux as a KVM guest OS
 #####################################
 
-To run |CL| in a virtualized environment using abbr:`KVM (Kernel-based
-Virtual Machine)`, follow these steps.
+This section explains how to run |CLOSIA| in a virtualized environment using
+abbr:`KVM (Kernel-based Virtual Machine)`.
 
-1. Enable `Intel® Virtualization Technology`_ (Intel® VT) and
+Install KVM
+===========
+
+#. Enable `Intel® Virtualization Technology`_ (Intel® VT) and
    `Intel®Virtualization Technology for Directed I/O`_ (Intel® VT-d) in the
    host machine’s BIOS.
 
-2. Log in and get root privilege on the host machine::
+#. Log in and get root privilege on the host machine:
+
+   .. code-block:: console
 
     $ sudo -s
 
-3. Install `QEMU-KVM` on the host machine. Below are some example distros.
+#. Install `QEMU-KVM` on the host machine. Below are some example distros.
+
+   * |CL|:
+
+     .. code-block:: console
 
-   * |CL|::
+        # swupd bundle-add desktop-autostart kvm-host
 
-     # swupd bundle-add desktop-autostart kvm-host
+   * Ubuntu 16.04 LTS Desktop:
 
-   * Ubuntu 16.04 LTS Desktop::
+     .. code-block:: console
 
-     # apt-get install qemu-kvm
+        # apt-get install qemu-kvm
 
-   * Mint 18.1 “Serena” Desktop::
+   * Mint 18.1 “Serena” Desktop:
 
-     # apt-get install qemu-kvm
+     .. code-block:: console
 
-   * Fedora 25 Workstation::
+        # apt-get install qemu-kvm
 
-     # dnf install qemu-kvm
+   * Fedora 25 Workstation:
 
-4. Download the latest pre-built Clear Linux _`KVM image` file from
+     .. code-block:: console
+
+        # dnf install qemu-kvm
+
+Create a virtual machine
+========================
+
+#. Download the latest pre-built Clear Linux _`KVM image` file from
    the _`image` directory.
 
-5. Uncompress the downloaded image::
+#. Uncompress the downloaded image
 
    .. code-block:: console
 
-      # unxz clear-<version number>-kvm.img.xz
+      unxz clear-<version number>-kvm.img.xz
 
-6. Download the :file:`OVMF.fd` file that provides UEFI support for
-   virtual machines from the _`image` directory.
+#. Download the :file:`OVMF.fd` file that provides UEFI support for
+   virtual machines from the _`images` directory.
 
-7. Download the sample _`QEMU-KVM launcher` script from the _`image` directory.
+#. Download the sample _`QEMU-KVM launcher` script from the _`image` directory.
 
-8. Make the script executable::
+#. Make the script executable
+
+   .. code-block:: console
 
-   # chmod +x start_qemu.sh
+      # chmod +x start_qemu.sh
 
-9. Start the |CL| KVM virtual machine::
+#. Start the |CL| KVM virtual machine:
 
-   # ./start_qemu.sh clear-<version number>-kvm.img
+     .. code-block:: console
+
+        # ./start_qemu.sh clear-<version number>-kvm.img
 
 #. Log in and set the root password.
 
-#. To SSH into the |CL| VM, follow these steps.
+#. To SSH into the |CL| VM, follow these steps:
```

Comment:
I put this info in a new sub-section

SSH into the new |CL| VM
----------------------------

You can do SSH login to your new |CL| VM. To do so, you need to launch the VM and follow these steps: